classes/site/Version.inc.php

```diff
@@ -284,6 +285,47 @@ function getVersionString($numeric = true) {
 
 		return $numericVersion;
 	}
+
+	/**
+	 * Checks if this version is compatible with given version
+	 * Returns:
+	 * 		true iff the version given is compatible with this version
+	 * 		false iff the version given is incompatible with this version
+	 * @param $version string/Version the version to compare against
```

Comment:
Is this intended as `string|Version`?  `VersionParser::parseConstraints()` expects a string, so specifying `Version` here seems suspect.

Reply:
No, this should just be string. The versionParser is removed.

Reply:
`@param $version string/Version` is still malformed.